src/main/hugo/content/howtos/development/building/_index.md

```diff
@@ -0,0 +1,26 @@
++++
+pre = ""
+title = "Build"
+weight = 5
+tags = ["build", "report", "IntelliJ", "Gradle", "Javadoc"]
+summary = "How to build the project"
++++
+
+## Building the project
```

Comment:
OK that's inconsistent.
The content of this how-to should only mention Gradle. In order:

1. gradlew vs gradlew.bat
2. build with `assemble`
3. test with `test`
4. complete QA with `check`
5. generation of the website with `hugoBuild`
6. preview of the website with `hugo --comand=serve`

Reply:
Changes integrated